config.md

```diff
@@ -289,14 +290,16 @@ For Windows based systems the user structure has the following fields:
 
 ## Platform-specific configuration
 
-[**`platform.os`**](#platform) is used to lookup further platform-specific configuration.
+[**`platform.os`**](#platform) is used to specify platform-specific configuration.
+Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
+Implementations MUST error out when invalid values are encountered and MUST generate a clear error message and exit non-zero when encountering valid values it chooses to not support.
```

Comment:
I think we want to drop “exit non-zero”, because `config.md` should be independent of the API used to invoke the runtime.  “generate a clear error message” is generic, and the relationship between error generation and exit-code should be [setup in the API docs for APIs which use exit-codes][1].

And “clear” is hard to test.  Maybe make it “MUST [generate an error][2]”?  Then we'd trust on runtime authors to make it clear.  If we don't trust runtime authors, we should say something enforceable, but that's hard to do when [we don't even require the runtime to expose error to the user][2].

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-0c1e5a5baa5649945697a530e09acbf5R66
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/runtime.md#errors